src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -60,14 +62,17 @@ const SubjectsStep = ({ stepLabel, btnsBox }) => {
   
   return (
     <Box sx={ styles.container }>
-      <Box sx={ styles.imgContainer }>
+      { isDesktop && (<Box sx={ styles.imgContainer  }>
```

Comment:
move this container with image to variable, dry

Reply:
fixed